jsapp/js/components/processing/processingBody.module.scss

```diff
@@ -107,6 +107,8 @@
   width: 100%;
   overflow-y: auto;
   flex: 1;
+  // TODO: have a 'full' option that turns off other displays
+  max-height: sizes.$x200;
```

Comment:
Didn't realize this affected the left :P I think we should still have some max height on the Transx displays on the sidebar though. If there are 2 or more bigger Transx displays it would push any other display off.

I think we would need:
1. A max height for the Transx displays
2. A limit to the amount displayed at once (that fits with the max height)
3. A way to "full screen" the display

Or we could overflow the entire sidebar for MVP?